doc_src/auto_examples/README.txt

```diff
@@ -0,0 +1,5 @@
+This is the Franck lab gallery
+==============================
+
+Below is a gallery of examples using our 
+various functions.
```

Comment:
@guinneak198 : did you create this file, or was it auto-generated?

Reply:
I created it sphinx needs  a py,rst,txt,zip for py and zip for jupyter, ipynb to work

Reply:
I don't understand this -- why do you need to create a txt file that looks just like the rst file?  Can you point me to the instructions that lead you to do this?

Similarly, I don't understand the need for the .zip file, etc.

Finally, when I looked at the documentation, I don't see any way of opening an ipynb file, so I'm a bit confused by the references to ipynb here.

Reply:
yes, it sounds like you are manually generating files that should be automatically generated for you -- see [here](https://sphinx-gallery.github.io/stable/getting_started.html) -- also that same page talks about setting up the thumbnail page.

Reply:
in their example, the directory they call `doc` we call `doc_src`

Reply:
you should not check in any of the automatically generated files on git

Reply:
we only check in files in the `doc` directory (the output of `make github`) -- but that is a **very** specific case:

- we need to do this b/c we are using github to display our documentation
- we should really only be doing this in an isolated commit -- we should really just have a commit that's a "documentation build" where we don't change any code or rst files, and just build the documentation.

Reply:
OK, so I've fixed all the stuff related to sphinx gallery